META.list

```diff
@@ -755,23 +755,23 @@ https://raw.githubusercontent.com/supernovus/perl6-fastcgi/master/META6.json
 https://raw.githubusercontent.com/supernovus/perl6-http-client/master/META6.json
 https://raw.githubusercontent.com/supernovus/perl6-http-easy/master/META6.json
 https://raw.githubusercontent.com/supernovus/perl6-http-status/master/META6.json
-https://raw.githubusercontent.com/supernovus/perl6-method-modifiers/master/META.info
+https://raw.githubusercontent.com/supernovus/perl6-method-modifiers/master/META6.json
 https://raw.githubusercontent.com/raku-community-modules/raku-mime-types/master/META6.json
 https://raw.githubusercontent.com/supernovus/perl6-text-table-list/master/META6.json
 https://raw.githubusercontent.com/supernovus/perl6-timezone/master/META6.json
-https://raw.githubusercontent.com/supernovus/perl6-web-app-ballet/master/META.info
+https://raw.githubusercontent.com/supernovus/perl6-web-app-ballet/master/META6.json
 https://raw.githubusercontent.com/supernovus/perl6-web-app-mvc/master/META6.json
 https://raw.githubusercontent.com/supernovus/perl6-web-template/master/META6.json
 https://raw.githubusercontent.com/supernovus/perl6-web/master/META6.json
 https://raw.githubusercontent.com/supernovus/perl6-xml-query/master/META6.json
 https://raw.githubusercontent.com/supernovus/template6/master/META6.json
-https://raw.githubusercontent.com/supernovus/temporal-utils/master/META.info
+https://raw.githubusercontent.com/supernovus/temporal-utils/master/META6.json
```

Comment:
This seems to be now at https://github.com/raku-community-modules/DateTime-Utils/blob/master/META6.json